src/shared/config.ts

```diff
@@ -50,3 +50,6 @@ export const instanceLinkRegex = new RegExp(
 );
 
 export const testHost = "0.0.0.0:8536";
+
+export const validActorRegexPattern =
+  "^\\w+|[\\p{Script=Arabic}\\d_]+|[\\p{Script=Cyrillic}\\d_]+$";
```

Comment:
`\w` is equivalent to `[a-zA-Z0-9_]`. `\d` is equivalent to `[0-9]`.

Reply:
In the backend we currently have three separate regexes for this, so you can also change it to this single regex.